docs/api/utils/derive.mdx

```diff
@@ -7,6 +7,13 @@ description: 'create a new proxy derived from others'
 
 ## `derive`
 
+<blockquote className="note">
+ℹ️ &nbsp; Note
+
+There is a new library called [valtio-reactive](https://github.com/valtiojs/valtio-reactive) that is a drop-in replacement for `derive` and provides a more powerful API. It is recommended to use that library instead.
```

Comment:
It's not a drop-in replacement. It's purpose and implementation design is a bit different.
`watch` will be deprecated in favor of `valtio-reactive`.

So, I'd suggest to remove this section for now.